tasks/main.yaml

```diff
@@ -56,6 +85,14 @@
         - item.type|d('') == 'custom'
         - logging_enabled|d(true)
 
+    - name: Install python selinux library for logging_debug
+      yum:
+        name: '{{ "python3-libselinux" if use_python3 else "libselinux-python" }}'
+        state: 'latest'
```

Comment:
package module should have been used here, not yum. https://github.com/oasis-roles/meta_standards/blob/master/README.md#ansible-best-practices